Nodejs/Product/Nodejs/SharedProject/SystemUtilities.cs

```diff
@@ -0,0 +1,142 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.NodejsTools.SharedProject
+{
+    /// <summary>
+    /// Utility for accessing window IShell* interfaces in order to use them to launch a process unelevated
+    /// </summary>
+    internal class SystemUtility
+    {
+        /// <summary>
+        /// We are elevated and should launch the process unelevated. We can't create the
+        /// process directly without it becoming elevated. So to workaround this, we have
+        /// explorer do the process creation (explorer is typically running unelevated).
+        /// </summary>
+        internal static void ExecuteProcessUnElevated(string process, string args, string currentDirectory = "")
+        {
+            var shellWindows = (IShellWindows)new CShellWindows();
+
+            // Get the desktop window
+            object loc = CSIDL_Desktop;
+            object unused = new object();
+            int hwnd;
+            var serviceProvider = (IServiceProvider)shellWindows.FindWindowSW(ref loc, ref unused, SWC_DESKTOP, out hwnd, SWFO_NEEDDISPATCH);
```

Comment:
I spent a while trying to read the docs to understand these interfaces, methods, and args, and.... 😕 

If this is taken from other working code, and you've tested it thoroughly, then seems OK. But it's not the most intuitive or readable code we've checked in.

Reply:
I agree, I tested the most common browsers and through different launch scenarios and everything looked good. 

The code has been taken from the Visual Studio installer and it's also the same code Diagnostics has recently adopted for this same Chrome issue.